modules/nuxt-feedme.yml

```diff
@@ -0,0 +1,16 @@
+name: nuxt-feedme
+description: The RSS feed module for Nuxt web framework
```

Comment:
From the looks of it, the module does more than an RSS feed, it also supports JSON and Atom. Does the description fit then? Maybe you mention the other feed types.

Reply:
> From the looks of it, the module does more than an RSS feed, it also supports JSON and Atom. Does the description fit then? Maybe you mention the other feed types.

To be honest, I'm not sure that this is required since RSS, ATOM and JSON is major types for feed representation. At the top of the [README.md](https://github.com/helltraitor/nuxt-feedme#configured-out-of-the-box-for-nuxt-content) file, default configuration shows all supported feed types.

I think that this is redundant, but, If you're still think so, I can add something like `... with ATOM, JSON and RSS support`

Reply:
Or something like "The universal feed module for Nuxt "

Reply:
> Or something like "The universal feed module for Nuxt "

Done, I can squash commits if needed